bwt/bwt_test.go

```diff
@@ -0,0 +1,265 @@
+package bwt
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"golang.org/x/exp/slices"
+)
+
+func ExampleBWT_Count() {
```

Comment:
Example tests should be in their own example_test.go file with the package name `bwt_test` so that package functions are called like `bwt.Count` to more closely emulate usage outside the package.

Reply:
Done